packages/python-teamcity-messages/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+source /opt/mesosphere/environment.export
+export LIB_INSTALL_DIR="$PKG_PATH/lib/python3.5/site-packages"
+mkdir -p "$LIB_INSTALL_DIR"
+
+pip3 install --no-deps --install-option="--prefix=$PKG_PATH" --root=/ /pkg/src/teamcity-messages
```

Comment:
same as above, please copy/paste the standard template: https://github.com/dcos/dcos/blob/master/packages/python-docopt/build#L6